connect/src/com/telenor/connect/ConnectSdk.java

```diff
@@ -45,6 +52,9 @@
     private static String sPaymentCancelUri;
     private static String sPaymentSuccessUri;
     private static SdkProfile sdkProfile;
+    private static ConnectivityManager connectivityManager;
+    private static Network cellularNetwork;
```

Comment:
I wrote a very long description of why this is safe at it is (in case of turning on/off the interfaces). Whoever is interested might read it.
It's in the design code doc, where the 
public WebResourceResponse fetchUrlTroughCellular(String originalUrl) is described.
The doc is at: https://confluence.skunk-works.no/confluence/display/Scale/Facilitating+Header+Enrichment+over+WiFi+in+MobileConnect+%28and+Connect+ID%29+SDK

I decided not to put those comments in the repo as they might expose some TelenorNorway operations stuff.